2019 카카오 개발자 겨울 인턴십/튜플/PGS_튜플_이지영.java

```diff
@@ -0,0 +1,39 @@
+import java.util.*;
+
+public class PGS_튜플_이지영 {
+
+    class Solution {
+        Map<Integer, Integer> map = new HashMap<>();
+
+        public int[] solution(String s) {
+            String[] tuples = s.substring(1, s.length()-1).split("},");
+
+            for (int i=0; i<tuples.length; i++) {
+                String tuple = tuples[i].substring(1);
+                if (i == tuples.length-1) tuple = tuple.substring(0, tuple.length()-1);
+
+                String[] nums = tuple.split(",");
+
+                for (String num : nums) {
+                    int n = Integer.parseInt(num);
+
+                    if(!map.containsKey(n)) {
+                        map.put(n, 0);
+                    }
+                    map.put(n, map.get(n)+1);
```

Comment:
```suggestion
                    map.put(n, map.getOrDefault(n,0)+1);
```
map.getOrDefault(key,defaultValue)을 이용하여 코드를 간결화 할 수 있을 것 같아요..!

Reply:
킹갓호 👍

Reply:
이렇게 편한 함수를 진호님만 알고 계셨던 거에요? 배신감 느끼고 갑니다...😭